src/pybind/ceph_argparse.py

```diff
@@ -1023,6 +1082,7 @@ def validate(args, signature, flags=0, partial=False):
                 if kwarg_desc:
                     validate_one(kwarg_v, kwarg_desc)
                     store_arg(kwarg_desc, d)
+                    matchcnt += 1
```

Comment:
This is really just a driveby fix, if you pass a long (--) paramenter using it's name in the CLI without param=true. Then it'll be checked positionally. The cmdict ends up correct and a with the right matches but because we don't increment the matchcnt, the sig being validated will fail.